main.go

```diff
@@ -76,25 +73,51 @@ func NewGame(g *Game) {
 	g.TileRenderer = renderer
 	g.LDTKProject = ldtkProject
 
+	level := g.LDTKProject.Levels[g.Level]
+
 	bg := ebiten.NewImage(
-		g.LDTKProject.Levels[g.Level].Width,
-		g.LDTKProject.Levels[g.Level].Height,
+		level.Width,
+		level.Height,
 	)
-	bg.Fill(g.LDTKProject.Levels[g.Level].BGColor)
+	bg.Fill(level.BGColor)
 
 	// Render map
-	g.TileRenderer.Render(g.LDTKProject.Levels[g.Level])
+	g.TileRenderer.Render(level)
 	for _, layer := range g.TileRenderer.RenderedLayers {
 		bg.DrawImage(layer.Image, &ebiten.DrawImageOptions{})
 	}
 	g.Background = bg
 
-	//Load sprites
+	// Create space for collision detection
+	g.Space = resolv.NewSpace(level.Width, level.Height, 16, 16)
```

Comment:
If we have any sprites that are bigger than 16×16px then we should increase the space's tile resolution to fit it.